pkg/placement/controller.go

```diff
@@ -112,18 +115,21 @@ func NewController(mgr ctrlm.Manager, wdsRestConfig *rest.Config, imbsRestConfig
 
 	ocmClient := *ocm.GetOCMClient(imbsRestConfig)
 
+	gvkGvrMapper := util.NewGvkGvrMapper()
+
 	controller := &Controller{
-		wdsName:          wdsName,
-		logger:           mgr.GetLogger(),
-		ocmClient:        ocmClient,
-		dynamicClient:    dynamicClient,
-		kubernetesClient: kubernetesClient,
-		extClient:        extClient,
-		listers:          make(map[string]*cache.GenericLister),
-		informers:        make(map[string]*cache.SharedIndexInformer),
-		stoppers:         make(map[string]chan struct{}),
-		gvksMap:          make(map[string]*schema.GroupVersionResource),
-		workqueue:        workqueue.NewRateLimitingQueue(ratelimiter),
+		wdsName:                   wdsName,
+		logger:                    mgr.GetLogger(),
+		ocmClient:                 ocmClient,
+		dynamicClient:             dynamicClient,
+		kubernetesClient:          kubernetesClient,
+		extClient:                 extClient,
+		listers:                   make(map[string]*cache.GenericLister),
+		informers:                 make(map[string]*cache.SharedIndexInformer),
+		stoppers:                  make(map[string]chan struct{}),
+		placementDecisionResolver: NewPlacementDecisionResolver(gvkGvrMapper),
+		gvkGvrMapper:              gvkGvrMapper,
```

Comment:
remove variable.. `gvkGvrMapper: util.NewGvkGvrMapper()`

Reply:
it needs to be the same mapper for both